google-cloud-datastore/src/main/java/com/google/cloud/datastore/DatastoreOptions.java

```diff
@@ -84,20 +94,50 @@ public static class Builder extends ServiceOptions.Builder<Datastore, DatastoreO
 
     private String namespace;
     private String databaseId;
+    private TransportChannelProvider channelProvider = null;
+    private CredentialsProvider credentialsProvider = null;
 
     private Builder() {}
 
     private Builder(DatastoreOptions options) {
       super(options);
       namespace = options.namespace;
       databaseId = options.databaseId;
+      this.channelProvider = options.channelProvider;
```

Comment:
Nit: the above 2 variables don't have "this." but the new ones do.

Reply:
thanks, fixed to have the above two to also have `this.` prefix